.csslintrc

```diff
@@ -1 +1 @@
---ignore=box-sizing,compatible-vendor-prefixes,known-properties,important,unqualified-attributes
+--ignore=box-sizing,compatible-vendor-prefixes,known-properties,important,unqualified-attributes,duplicate-background-images
```

Comment:
This is actually just to avoid a bug in `jshint`... it appears as though very large background images (we're using data URIs) get truncated when they're doing a comparison so it _thinks_ they're the same.